airflow/utils/db.py

```diff
@@ -90,7 +90,7 @@
     "2.7.0": "405de8318b3a",
     "2.8.0": "10b52ebd31f7",
     "2.8.1": "88344c1d9134",
-    "2.9.0": "1fd565369930",
+    "2.9.0": "1949afb29106",
```

Comment:
Why'd this change?

Reply:
It's automatically retrieved by pre-commit - and for 2.9.0 it will keep on changing every time we update migrations.

Reply:
(until we release 2.9.0 that is).

Reply:
(it's also triggered when utils.db changes itself)

Reply:
The value has to match what's in the migration files. I guess that this is from a conflict resolution issue

Reply:
I get all that, but why didn't pre-commit force it to change in another PR then? I don't see how this PR would cause the head to move.

Reply:
Ah, @ephraimbuddy nailed it, pre-commit is changing it back to the prior value. So yeah bad conflict fix.

Reply:
```suggestion
    "2.9.0": "1fd565369930",
```

Then static checks will pass.

Reply:
I was working on a different PR, and this is the revision of the not-indexed migration file. We can improve this check later by considering only indexed files.